app/assets/stylesheets/views/_search.scss

```diff
@@ -133,6 +133,24 @@ main.search {
         float: left;
         width: $two-thirds;
       }
+
+      mark {
+        background-color: rgba(255, 255, 140, 0.5);
```

Comment:
The colour contrast between the link text and the yellow background isn't great:
![image](https://cloud.githubusercontent.com/assets/355079/9656162/393f0b10-522e-11e5-9749-c9e74cc6ac32.png)

We've been using `$yellow-25` on the Digital Marketplace, which doesn't show up on high-contrast screens (the telly by the purple sofas for example).
![image](https://cloud.githubusercontent.com/assets/355079/9656181/5a9095d6-522e-11e5-90fd-42712ae22b14.png)

We don't highlight search result page titles at the moment, but the tradeoff between keeping link text legible and having highlights visible is the tradeoff when you do.